testutils/collection/data.go

```diff
@@ -375,6 +375,18 @@ var (
 			ProfileId:        "1106b972-66ef-11e7-b172-db03f3689c9c",
 			SnapshotId:       "3769855c-a102-11e7-b772-17b880d2f537",
 		},
+		{
```

Comment:
can existing be re-used?

Reply:
When we run the test for create then the status of the structure changes to "creating", same with delete it changes to "deleting", so rather than using the same struct across we can use a copy for those 2 different test cases, else when we run test on all test cases together it fails, saying status should be availble not creating/deleting